### PR TITLE
Added a link to the github wiki tutorial for each guild, and added th…

### PR DESCRIPTION
**Pull request by jayRyan24**
…e t2 tutorial wiki link for each guild.

### CODE REVIEW

File: profiles/SampleMoonMage-setup.yaml [hunk 1]
```diff
@@ -5,6 +5,13 @@
 # 3 Lores: Appraisal, Tactics, Mechanical Lore (Does not include Scholarship)
 # 2 Additional: Astrology, Scholarship
 
+# High-level example YAMLS:
+#  - Jonas
```
Comment (rpherbig) [inline profiles/SampleMoonMage-setup.yaml:9]: Isn't `Jonas` also in the Necro high-level examples?

File: profiles/SampleTrader-setup.yaml [hunk 2]
```diff
@@ -12,6 +12,12 @@
 # You do not need to train any Magic to circle
 # However, you will probably end up training them all since they each have very useful abilities
 
+# High-level example YAMLS:
+#  - None right now!
```
Comment (rpherbig) [inline profiles/SampleTrader-setup.yaml:16]: Maybe add an ask for submissions?

**Comment by rpherbig** [comment]
Some (most?) of these tutorials don't exist - are you planning on adding them?

File: profiles/SampleMoonMage-setup.yaml (hunk 1 above)
Comment (jayRyan24) [reply profiles/SampleMoonMage-setup.yaml:9]: Yeah, that's why he's added!

Comment (jayRyan24) [reply profiles/SampleMoonMage-setup.yaml:9]: Oh, wait, shit. I meant Crannach... too many alts! Thanks, good catch!

File: profiles/SampleTrader-setup.yaml (hunk 2 above)
Comment (jayRyan24) [reply profiles/SampleTrader-setup.yaml:16]: Sure, will do that.

**Comment by jayRyan24** [comment]
Yes, they are being added.

**Comment by jayRyan24** [comment]
They *will* be there. I will finish them this weekend. I added/fixed/updated a few others this week.

**Pull request merged**